tests/PCTCompile/test50/src/test2.i

```diff
@@ -0,0 +1 @@
+DEFINE VARIABLE cTest AS CHARACTER INIT {src\test1.i} NO-UNDO.
```

Comment:
Still a backslash here, didn't see it at the first time, but unit tests failed again !